tokenserver/assignment/sqlnode/sql.py

```diff
@@ -384,6 +417,7 @@ def update_user(self, service, user, generation=None, client_state=None,
                 'nodeid': nodeid, 'generation': generation,
                 'keys_changed_at': keys_changed_at,
                 'client_state': client_state, 'timestamp': now,
+                'migration_state': ''
```

Comment:
I don't really understand what "migration_state" is for, but I found myself expecting this to use the passed-in value rather than resetting to the empty string.

Reply:
Ah, yeah, that's a bug. Thanks!